WcaOnRails/spec/mailers/previews/delegate_status_change_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/wca_id_claim_mailer
+class DelegateStatusChangeMailerPreview < ActionMailer::Preview
+  def notify_board_and_wqac_of_delegate_status_change
+    ActiveRecord::Base.transaction do
```

Comment:
I don't think we need a transaction here. If you copied this from another preview, I'd guess that that method was trying to temporarily create some state in the database for the preview? However, the current implementation here doesn't create anything in the database, so you shouldn't need a transaction.

Reply:
Yes, I copied it and adapted it, I will change this.